rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.retrievers/20_knn_retriever.yml

```diff
@@ -1,7 +1,7 @@
 setup:
   - skip:
-      version: ' - 8.11.99'
-      reason: 'kNN retriever added in 8.12'
+      version: ' - 8.12.99'
+      reason: 'classic retriever added in 8.12'
```

Comment:
should the reason be about `8.13` version, may be we can say `retrievers were added in 8.13`?

Reply:
Good catch! Fixed.